doc/branching_model.puml

```diff
@@ -3,39 +3,37 @@
 
 == Release ==
 participant "Dev branch" as devbranch order 10 #LightBlue
-participant "RC branch" as rcbranch order 20 #YellowGreen
+participant "Release branch" as rlsbranch order 20 #YellowGreen
 participant "Master branch" as master order 30 #99FF99
 participant "Hotfix branch" as hotbranch order 40 #DarkSalmon
 
-{startrc} devbranch -> rcbranch ++ #YellowGreen: Start release x.y.0
+{startrc} devbranch -> rlsbranch ++ #YellowGreen: Create branch release_x.y.0
 
 
 note left of devbranch
   Evolutions
 end note
 
 
-rcbranch <- devbranch: Merge bug fixes
-rcbranch <- devbranch: Merge more bug fixes
+rlsbranch <- devbranch: Merge bug fixes
+rlsbranch <- devbranch: Merge more bug fixes
 
 
-note right of rcbranch
+note right of rlsbranch
   - Bug fixes
-  - Tag "x.y.0-rc" on RC branch
```

Comment:
I've put the tag there due to the release-drafter. When you publish the notes, it creates a tag for you whether you want it or not. So in reality, there is a tag.

Reply:
OK so for any fix commited to the release branch, release drafter will automatically update the release info and create a tag? then it makes sense to keep that "anonymous" rc.

Reply:
No, release-drafter only watches `master` and `dev` right now. So it's only on the first occurrence. For now, the backport activities would have to be done manually. Need more testing of release-drafter there eventually, or just keep it manual if that's a low activity?